PlottingScripts/Basic_Plot.py

```diff
@@ -0,0 +1,128 @@
+"""This is a basic script for plotting an analysis file."""
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+import pdb
+
+from bokeh.plotting import figure, show, output_file
+
+
+"""This script is just the first step in visualizing the data set. This file
+comprises of the method creating an html bokeh plot and a command at the end to
+implement the method
+"""
+
+
+def makeplot(Dataframe, Cols, minvalues):
+    """Basic method to plot sensitivity anlaysis.
+
+    This is the method to generate a bokeh plot followung the burtin example
+    template at the bokeh website. For clarification, parameters refer to an
+    output being measured (Tmax, C, rxn2, etc.) and stats refer to the 1st or
```

Comment:
In the functions that I've written to read and process all our input data I've updated the parameters to say "k2" instead of "rxn2" to be more specific about what was adjusted.  We should update this throughout our code and documentation.